debian/rules

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/make -f
 # -*- makefile -*-
-VERSION := $(shell mvn org.apache.maven.plugins:maven-help-plugin:2.1.1:evaluate -Dexpression=project.version | grep -v "\[")
+VERSION := $(shell awk 'f{print;f=0} /<artifactId>cloudstack/{f=1}' pom.xml| sed -e 's/<[^>]*>//g'  -e 's/^[ \t]*//')
```

Comment:
Looks like this will be faster, though the previous version is more understandable so I would prefer to keep the previous one. We don't build packages often and it's usually Jenkins doing the heavy lifting.